nimibook.nimble

```diff
@@ -14,27 +14,32 @@ requires "nimib >= 0.1.2"
 requires "jsony >= 1.0.1"
 
 import os
+
 task genbook, "build book":
   selfExec(" r -d:release genbook.nim")
 
 task dumpbook, "dump book.json":
   selfExec(" r -d:release -d:dumpBook genbook.nim")
 
 task cleanbook, "remove all files created during build":
+  # Since assets now gets populated, I think we can just delete everything in docs
```

Comment:
Me and @pietroppeter discussed this at the bottom of #12 that we perhaps shouldn't delete the entire `docs/` folder but only the relevant files. Otherwise, it's hard to incorporate static (non-Nimib generated) files such as static images or stylesheets into the book. We'd have to create some way of adding user-assets to the `docs/` folder in that case like you did for the built-in assets if we are going the `rm docs` path.